src/style.css

```diff
@@ -0,0 +1,129 @@
+* {
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+
+main {
+  width: 60%;
+  margin: 10em auto;
+  display: flex;
+  flex-direction: column;
+  gap: 3em;
+}
+
+main h1 {
+  text-align: center;
+}
+
+main ul {
+  display: flex;
+  flex-direction: column;
+  box-shadow: 0 4px 6px 3px #ebebeb;
+  border-radius: 3px;
+}
+
+main ul li {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1em 1.5em;
+  min-height: 60px;
+  font-size: 16px;
+}
+
+/* Linea inferior al elemento hijo li */
+main ul li:not(:last-child) {
+  border-bottom: 0.5px solid #e9e9e9;
+}
+
+/* Clear all */
+
+main ul li.list-footer {
+  background-color: #f8f8f8;
+  display: flex;
+  justify-content: center;
+  color: #a0a0a0;
+  cursor: pointer;
+}
+
+main ul li.list-footer:hover {
+  text-decoration: underline;
+  color: #202020;
+}
+
+/* End Clear all */
+
+/* Cursor mode */
+
+main ul li i {
+  cursor: pointer;
+  color: #a0a0a0;
+}
+
+main ul li i.fa-bars:hover {
+  cursor: move;
+}
+
+/* End cursor mode */
+
+button.trash {
+  background-color: transparent;
+  border: transparent;
+}
+
+main ul li i:hover {
+  color: #202020;
+}
+
+main ul li form {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  margin: -1em auto;
+  position: relative;
+}
+
+main ul li form input {
+  width: 100%;
+  border: none;
+  outline: none;
+  padding: 1em 0;
+  height: 40px;
+  font-size: 16px;
+}
+
+main ul li form button {
+  position: absolute;
+  right: 0;
+  background-color: transparent;
+  border: none;
+  padding: 0;
+}
+
```

Comment:
you applied a good style and UI design for this project. it looks awesome. 
I recommend you to use html class instead chaining the elements from its parent to itself, 🆗 
instead main ul li from button 😒  , you can use a class name for button ( ex. submit-btn), then you can access it easily by writing .submit-btn { // styles }. 

It is a good practice to use CSS selectors and classes. 😃  Happy coding!!!

Reply:
> Correction

Fixed unnecessary use of inheritance in css file

Reply:
Your use of ES-6 modules and adherence to best practices in JavaScript coding is greatly appreciated and truly elevates the quality of your work. Well done!"